scripts/devel/generate_devel_script.py

```diff
@@ -39,12 +39,36 @@
 from ros_buildfarm.templates import expand_template
 
 
+class _BuildToolArgsHelper(argparse.Action):
```

Comment:
Maybe move into a module to deduplicate?

Reply:
0d457c2